modules/util/portal-tools-service-builder/src/main/resources/com/liferay/portal/tools/service/builder/dependencies/service.ftl

```diff
@@ -103,9 +102,7 @@ public interface ${entity.name}${sessionTypeName}Service
 			, CTService<${entity.name}>
 		</#if>
 
-		<#if entity.isPermissionedModel()>
-			, PermissionedModelLocalService
-		<#elseif entity.isResourcedModel()>
+		<#if entity.isResourcedModel()>
```

Comment:
@dantewang @ChiZelin 
Any change on service builder need to make sure it works well for all other verisons, since we only have one version of tool.

Tina.